Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM usgswma/python:3.8
 
+RUN apt-get update
```

Comment:
I think this is fine - I wonder if this is something we can do in the docker image we pull on line 1: `usgswma/python:3.8` - since we're not installing anything before running the update.  It's definitely out of scope for this work, but could be another JIRA task?